mlrun/model.py

```diff
@@ -48,6 +49,14 @@ def _verify_dict(param, name, new_type=None):
             return new_type.from_dict(param)
         return param
 
+    @staticmethod
+    def _verify_str(param, name):
+        if param is not None and not isinstance(param, str):
+            raise mlrun.errors.MLRunInvalidArgumentError(
+                f"Parameter {name} must be a string. got {type(param)}"
```

Comment:
Should capitalize the first word in a sentence.
```suggestion
                f"Parameter {name} must be a string. Got {type(param)}"
```